nuxt.config.js

```diff
@@ -52,7 +52,7 @@ export default {
 
   // Content module configuration: https://go.nuxtjs.dev/config-content
   content: {
-    fullTextSearchFields: ['id', 'slug', 'title', 'author', 'thumbnail', 'category']
+    fullTextSearchFields: ['title', 'description', 'slug', 'text']
```

Comment:
Here you should add all the keys that you have in the YAML frontmatter, so also e.g. `contact` and `doi` (if you want to search that..). And since `contact` is a nested property, you should also add that

Reply:
> Here you should add all the keys that you have in the YAML frontmatter, so also e.g. `contact` and `doi` (if you want to search that..). And since `contact` is a nested property, you should also add that

Cool! The way to add nested property was exactly what I was looking for. I tried adding `contact` directly but Nuxt will complain `TypeError: Unsupported field type for full text search.` and the app will break. Thanks for the tip!